lib/pdf-editor-view.js

```diff
@@ -46,6 +46,11 @@ export default class PdfEditorView extends ScrollView {
     this.maxPageWidth = 0;
     this.toScaleFactor = 1.0;
 
+    this.nightMode = atom.config.get('pdf-view.nightMode');
+    if (this.nightMode) {
```

Comment:
Would be neat if we could also remove the night mode class if night mode was disabled in the settings. Right now, we only add the night mode if the setting is enabled, so there's no way to remove it without reloading/restarting. What do you think? 💭